java/values.yaml

```diff
@@ -17,10 +17,15 @@ readinessDelay: 30
 readinessTimeout: 3
 readinessPeriod: 15
 livenessPath: '/health/liveness'
-livenessDelay: 30
+livenessDelay: 0
```

Comment:
should readinessDelay also be set to 0?

Reply:
Agreed as shouldn't need this with startup probe introduction. Updated.